coops-ui/src/components/common/OfficeAddresses.vue

```diff
@@ -103,7 +103,7 @@
             />
           </div>
 
-          <div v-if="!isSame(registeredAddress, recordsAddress) || !inheritRegisteredAddress">
```

Comment:
Now this code is reactive, whereas the new code on line 289 isn't. (See my other comment.)